### ISSUE TITLE
aria-valuenow should not fail aria-required-attr when there is a fallback value

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1501]
The following shouldn't fail:

```html
<input type="range" role="slider" />
```

There are similar cases. Basically any element that has a value (see `commons.text.formControlValue`) should be allowed as an alternative for `aria-requirednow`.

**Comment by straker** [comment]
Should an empty value fail when the input is not populated?

```
<input type="number" role="spinbutton" <="" div="">
```

**Comment by chandana7393** [comment]
Tested, working as expected.
![no_violations](https://user-images.githubusercontent.com/45285854/58951276-4d276f00-87ae-11e9-8fcc-efec77422a14.PNG)

**Tested Environment:**
Axe-coconut - 3.8.0.15604v
Chrome - 74v
OS - Windows 10 64 bit.
